src/app/[locale]/specs/sync/page.mdx

```diff
@@ -0,0 +1,171 @@
+
+In most cases the repository data synchronized over the firehose is self-certifying (contains verifiable signatures), and consumers can verify content without making additional requests directly to account PDS instances. It is possible for services to redact events from the firehose, such that downstream services would not be aware of new content.
+
+Identity and account information is *not* self-certifying, and services may need need to verify independently. This usually means independent DID and handle resolution. Account hosting status might also be checked at account PDS hosts, to disambiguate hosting status at different pieces of infrastructure.
+
+The event message types are declared in the `com.atproto.sync.subscribeRepos` Lexicon schema, and are summarized below. A few fields are the same for all event types (except for `repo` vs `did` for `#commit` events):
+
+- `seq` (integer, required): used to ensure reliable consumption, as described in Event Streams
+- `did` / `repo`(string with DID syntax, required): the account/identity associated with the event
+- `time` (string with datetime syntax, required): an informal and non-authoritative estimate of when event was received. Intermediate services may decide pass-through this field as-is, or update to the current time
+
+### `#identity` Events
+
+Indicates that there *may* have been a change to the indicated identity (meaning the DID document or handle), and optionally what the current handle is. Does not indicate what changed, or reliably indicate what the current state of the identity is.
+
+Event fields:
+
+- `seq` (integer, required): same for all event types
+- `did` (string with DID syntax, required): same for all event types
+- `time` (string with datetime syntax, required): same for all event types
+- `handle` (string with handle syntax, optional): the current handle for this identity. May be `handle.invalid` if the handle does not currently resolve correctly.
+
+Presence or absence of the `handle` field does not indicate that it is the handle which has changed.
+
+The semantics and expected behavior are that downstream services should refresh their identity data (resolve the DID and handle) for the indicated DID. 
+
+Identity events are emitted on a "best-effort" basis. Is is possible for the DID document or handle resolution status to change without any atproto service detecting the change, in which case an event would not be emitted. It is also possible for the event to be emitted redundantly, when nothing has actually changed.
+
+Intermediate services (eg, relays) may chose to modify or pass through identity events:
+
+- they may replace the handle with the result of their own resolution; or always remove the handle field; or always pass it through unaltered
+- they may filter out identity events if they observe that identity has not actually changed
```

Comment:
not sure these are good ideas, haha.  just as a framing/psychological nit, it might be better to rephrase these as "things you can't assume intermediate services aren't doing to you" rather than as "things you can do to other participants as an intermediate service"?

Reply:
As an example, we think it is likely a best practice for relays to de-duplicate rapidly repeated and redundant identity events on the firehose.